packages/home-spa/src/styles/_colors.scss

```diff
@@ -0,0 +1,94 @@
+:root {
+    // Gray
+    --op-white: #FFFFFF;
+    --op-gray-100: #F3F3F3;
+    --op-gray-200: #DEDEDE;
+    --op-gray-300: #C9C9C9;
+    --op-gray-400: #AAAAAA;
+    --op-gray-500: #8B8B8B;
+    --op-gray-600: #6C6C6C;
+    --op-gray-700: #4D4D4D;
+    --op-gray-800: #2E2E2E;
+    --op-black: #000000;
+    
+      /*  Red  */
+    --op-red-100: #FDDBDB;
+    --op-red-200: #FAB6B6;
+    --op-red-300: #F56D6D;
+    --op-red-400: #EE0000;
+    --op-red-500: #BE0000;
+    --op-red-600: #8F0000;
+    --op-red-700: #5F0000;
+    
+      /*  Teal  */
+    --op-teal-100: #E4F3F4;
+    --op-teal-200: #AEDCDD;
+    --op-teal-300: #79C4C6;
+    --op-teal-400: #43ADAF;
+    --op-teal-500: #368A8C;
+    --op-teal-600: #286869;
+    --op-teal-700: #1B4546;
+    
+     /*  indigo  */
+    --op-indigo-100: #E2EAF6;
+    --op-indigo-200: #A7C0E4;
+    --op-indigo-300: #6C97D3;
+    --op-indigo-400: #316DC1;
+    --op-indigo-500: #27579A;
+    --op-indigo-600: #1D4174;
+    --op-indigo-700: #142C4D;
+    
+    /*  Green */
+    --op-green-100: #EDF6F0;
+    --op-green-200: #C9E5D3;
+    --op-green-300: #A6D3B6;
+    --op-green-400: #82C299;
+    --op-green-500: #689B7A;
+    --op-green-600: #4E745C;
+    --op-green-700: #344E3D;
+    
+    /*   Blue */
+    --op-blue-100: #DBF0F7;
+    --op-blue-200: #93D3E6;
+    --op-blue-300: #4CB6D6;
+    --op-blue-400: #0499C5;
+    --op-blue-500: #037A9E;
+    --op-blue-600: #025C76;
+    --op-blue-700: #023D4F;
+    
+    /*   Purple */
+    --op-purple-100: #E8E4F6;
+    --op-purple-200: #BAADE3;
+    --op-purple-300: #8C77D1;
+    --op-purple-400: #5E40BE;
+    --op-purple-500: #4B3398;
+    --op-purple-600: #382672;
+    
+    /* Orange */
+    --op-orange-100: #FDDBC7;
+    --op-orange-200: #FAB890;
+    --op-orange-300: #F7823C;
+    --op-orange-400: #C65E2B;
+    --op-orange-500: #944620;
+    --op-orange-600: #632F16;
+    
+    /*   Yellow - orange */
+    --op-yellow-orange-100: #FEE7CD;
+    --op-yellow-orange-200: #FCCE9A;
+    --op-yellow-orange-300: #FAAA4F;
+    --op-yellow-orange-400: #C8883F;
+    --op-yellow-orange-500: #96662F;
+    
+    /*   Yellow - green */
+    --op-yellow-green-100: #F2F7E6;
+    --op-yellow-green-200: #D9E6B4;
+    --op-yellow-green-300: #C0D682;
+    --op-yellow-green-400: #9AAB68;
+    --op-yellow-green-500: #73804E;
+    
+    /*   Yellow */
+    --op-yellow-100: #FFF7C4;
+    --op-yellow-200: #FFEA6C;
+    --op-yellow-300: #CCBB56;
+    --op-yellow-400: #998C41;
+}
```

Comment:
Can things like these (colors, styles, mixins, components) be please moved to a new packages/common directory so that all apps and microservices can reuse these?

Reply:
The packages/common directory is a good option and the only flaw I see is that it would be too vague as to what all is common, is it just for SPAs or is it also for microservices